test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smdataparallel.py

```diff
@@ -100,10 +121,8 @@ def test_smdataparallel_throughput(instance_types, n_virginia_ecr_image, py_vers
     """
     Tests smddprun throughput
     """
-    _, image_framework_version = get_framework_and_version_from_tag(n_virginia_ecr_image)
-    image_cuda_version = get_cuda_version_from_tag(n_virginia_ecr_image)
-    if Version(image_framework_version) < Version("2.4.1") or image_cuda_version != "cu110":
-        pytest.skip("EFA is only supported on CUDA 11, and on TensorFlow 2.4.1 or higher")
+    validate_or_skip_smdataparallel(n_virginia_ecr_image)
```

Comment:
Why run the less restrictive validation at all when the more restrictive validation for `efa` filters out more cases?

Reply:
you are right, will remove validate_or_skip_smdataparallel/validate_or_skip_smmodelparallel for all efa tests